consensus/polybft/stake_manager.go

```diff
@@ -115,12 +121,23 @@ func (s *stakeManager) PostBlock(req *PostBlockRequest) error {
 		return err
 	}
 
-	s.logger.Debug("Stake manager on post block", "block", req.FullBlock.Block.Number(),
-		"last saved", fullValidatorSet.BlockNumber, "last updated", fullValidatorSet.UpdatedAtBlockNumber)
+	s.logger.Debug("Stake manager on post block",
+		"block", req.FullBlock.Block.Number(),
```

Comment:
Nitpick: extract `blockNumber` to a local var and reuse it:
`blockNumber := req.FullBlock.Block.Number()`

Reply:
fixed https://github.com/0xPolygon/polygon-edge/pull/1928/commits/7459127e696d97266ca729acc9ffa7de07762236